rootfs-builder/ubuntu/config.sh

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2017 Intel Corporation
```

Comment:
This is a new file so should show 2018 but also your name/or company right @grahamwhaley ? Same comment for the other new files.